### PR TITLE
feat: add book autofill function

### PR DESCRIPTION
**Pull request by kdvuong**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ android {
 
     defaultConfig {
         applicationId "com.cmput301f20t21.bookfriends"
-        minSdkVersion 24
```
Comment (c25vdw) [inline app/build.gradle:14]: where do we need this change? just curious tho.

**Review by c25vdw — APPROVED** [review]
lgtm

File: app/build.gradle (hunk 1 above)
Comment (kdvuong) [reply app/build.gradle:14]: String.join() lol

Comment (kdvuong) [reply app/build.gradle:14]: For joining author name when a book has multiple authors

**Review by zhpeng811 — APPROVED** [review]
lgtm

**Comment by c25vdw** [comment]
Congrats on the 100th PR!!!!!!! :100: :tada: :1st_place_medal: :partying_face: :100: :100: :100:

**Pull request merged**